test/CreateGraphTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Islandora\Chullo;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use Islandora\Chullo\FedoraApi;
+
+class CreateGraphTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @covers  Islandora\Chullo\FedoraApi::createGraph
+     * @uses    GuzzleHttp\Client
+     */
+    public function testReturnsTrueOn204()
+    {
+        $mock = new MockHandler([
+            new Response(204),
+        ]);
+
+        $handler = HandlerStack::create($mock);
+        $guzzle = new Client(['handler' => $handler]);
+        $api = new FedoraApi($guzzle);
+
+        $result = $api->createGraph(new \EasyRdf_Graph());
```

Comment:
Good question.  I'm thinking it's ok to create without content because it would be like an empty post.  But if the graph is empty we should probably just be passing NULL into `createResource()`.  I don't actually know what a serialized empty graph in turtle will return.  A blank string?  Some prefix declarations?